### PR TITLE
bugfix:  input labelPlaceholder not center

### PR DESCRIPTION
**Pull request by KieSun**
## 📝 Description

Input `labelPlaceholder` not center. If I use `placeholder` props, the text is center, but `labelPlaceholder` not. I think them behavior should be same.

## ⛳️ Current behavior (updates)
before

## 🚀 New behavior

image

## 💣 Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Comment by KieSun** [comment]
@jrgarciadev

File: packages/react/src/input/input.styles.ts [hunk 1]
```diff
@@ -567,7 +566,8 @@ export const StyledInputBlockLabel = styled('label', {
       underlined: true,
       asPlaceholder: true,
       css: {
-        left: '$space$2'
+        left: '$space$2',
+        top: '20%'
```
Comment (tianenpang) [inline packages/react/src/input/input.styles.ts:570]: The `labelPlaceholder` is still a bit out of position (vertically) when it is `undelined`.

pr602-underline-label-placeholder

**Review by tianenpang — CHANGES_REQUESTED** [review]
Hi @KieSun thank you for contributing, I left some comments.

File: packages/react/src/input/input.styles.ts (hunk 1 above)
Comment (KieSun) [reply packages/react/src/input/input.styles.ts:570]: Do you mean they all need to be centered?

Comment (tianenpang) [reply packages/react/src/input/input.styles.ts:570]: Yep, it looks like all the Inputs that use `labelPlaceholder` have a shift of around 5 pixels up.

Hi @jrgarciadev is that a feature? 🤔

Comment (jrgarciadev) [reply packages/react/src/input/input.styles.ts:570]: Hey guys sorry for the delay, that was done on purpose due to the top animation

**Pull request closed without merging**